backend/routes/collections.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const router = express.Router();
+const { Collection } = require('../data').models;
+
+router.use(express.json());
+
+router.get('/:businessId/', (req, res, next) => {
```

Comment:
also non-blocking but i would strongly prefer the route `GET /api/business/:id/collections` instead of `GET /api/collections/:businessId`